src/mlpack/core/data/load_csv.hpp

```diff
@@ -114,16 +114,15 @@ class LoadCSV
         // Extract the number of columns.
         auto findColSize = [&cols](iter_type) { ++cols; };
         qi::parse(line.begin(), line.end(),
-            stringRule[findColSize] % delimiterRule);
+                  stringRule[findColSize] % delimiterRule);
       }
 
       // I guess this is technically a second pass, but that's ok... still the
       // same idea...
       if (MapPolicy::NeedsFirstPass)
       {
         // In this case we must pass everything we parse to the MapPolicy.
-        auto firstPassMap = [&](const iter_type& iter)
-        {
+        auto firstPassMap = [&](const iter_type& iter) {
```

Comment:
The opening brace should be on the following line here, not moved to be on the same line.